src/app.js

```diff
@@ -49,6 +53,24 @@ app.get('/filmer/:movieId', async (req, res) => {
     }
   }
 });
+app.get('/api/movies/:movieId', (req, res) => {
+  const { movieId } = req.params;
+  res.render('reviews', { movieId });
+});
+
+app.post('/api/movies/:movieId/reviews', async (req, res) => {
+  const movieId = req.params.movieId;
+  const { author, comment, rating } = req.body;
+
+  const newReview = {
+    movie: movieId,
+    comment,
+    rating,
+    author,
+  };
+  await review(newReview);
```

Comment:
I find the name of this function to be very ambiguous. Because I know the assignment, I can guess what it does, but I would have still used a more explicit name, like `sendReview()`, `submitReview()` or something like that.

And the best thing of course would be to make it a method on an API adapter. That way, it can more easily be mocked and tested.